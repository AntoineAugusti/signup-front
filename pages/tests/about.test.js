import ReactTestRenderer from 'react-test-renderer'
import About from '../../pages/about.html'

describe('pages | about', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<About />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
