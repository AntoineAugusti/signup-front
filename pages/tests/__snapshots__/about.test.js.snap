// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages | about render should match snapshot 1`] = `
<div
  className="page"
>
  <header>
    <nav
      className="nav"
    >
      <div
        className="nav__container"
      >
        <a
          className="nav__link"
          href="https://api.gouv.fr/"
        >
          <img
            alt="Accueil de signup.api.gouv.fr"
            className="nav__logo"
            src="https://api.gouv.fr/img/logo_api.svg"
          />
        </a>
        <ul
          className="nav__links"
        >
          <li>
            <a
              href="http://localhost:3022/oauth/authorize?client_id=82546188522214c3577d35c283ce8842786649b35a026a9d44908037a597f29b&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Foauth-callback&scope=&response_type=token&state="
            >
              Se connecter
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </header>
  <main>
    <div>
      
      <section
        className={null}
        id={null}
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="column"
            >
              <h2
                className="section__title"
              >
                Ne plus demander de pièces justificatives inutiles
              </h2>
              <p>
                Limiter les demandes des pièces justificatives ou informations lorsqu’elles sont détenues par d’autres administrations ( CAF, DGFiP, Pôle Emploi…).
              </p>
              <p>
                Limiter le risque de fraude documentaire en récupérant des informations certifiées à la source dans un point d‘accès unique.
              </p>
            </div>
            <div
              className="column"
            >
              <h2
                className="section__title"
              >
                Accéder aux données de façon sécurisée
              </h2>
              <p>
                L‘accès aux données est réalisé au travers de protocoles de communications sécurisés et seuls les opérateurs agréés peuvent se connecter à l‘API.
              </p>
              <p>
                Pour obtenir un agrément, vous devez justifier d‘une simplification pour les citoyens, et vous engager à n‘accéder aux données personnelles qu‘avec l‘accord explicite de l‘usager.
              </p>
              <a
                className="button"
                href="../static/docs/charte.pdf"
                onClick={[Function]}
              >
                Consultez la charte
              </a>
            </div>
          </div>
        </div>
      </section>
      <section
        className={null}
        id={null}
      >
        <h2
          className="section__title"
        >
          Comprendre la différence avec FranceConnect
        </h2>
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="column"
            >
              <p>
                API Particulier et FranceConnect sont deux systèmes indépendants mais complémentaires.
              </p>
              <table
                className="table-fc"
              >
                <tr>
                  <th />
                  <th>
                    FranceConnect
                  </th>
                  <th>
                    API Particulier
                  </th>
                </tr>
                <tr>
                  <th>
                    Nature
                  </th>
                  <td>
                    Fournisseur d’Identité et tiers de confiance
                  </td>
                  <td>
                    Fournisseur de Données
                  </td>
                </tr>
                <tr>
                  <th>
                    Identification
                  </th>
                  <td>
                    Citoyen et Fournisseur de Service
                  </td>
                  <td>
                    Fournisseur de Service
                  </td>
                </tr>
                <tr>
                  <th>
                    Type de donnée
                  </th>
                  <td>
                    liée à la personne
                  </td>
                  <td>
                    liée à la personne, au foyer familial, aux parents et aux enfants
                  </td>
                </tr>
              </table>
              <p>
                API Particulier et FranceConnect peuvent être utilisés seul ou conjointement.
              </p>
            </div>
            <div
              className="column"
            >
              <img
                alt="Logo du service FranceConnect"
                src="../static/images/illustrations/logo-fc.png"
              />
            </div>
          </div>
        </div>
      </section>
    </div>
  </main>
  <footer
    className="footer"
  >
    <div
      className="container"
    >
      <div
        className="jsx-696190668 footer__logo"
      >
        <a
          className="jsx-696190668"
          href="https://api.gouv.fr/"
        >
          <img
            alt="Logo api.gouv.fr"
            className="jsx-696190668"
            src="/static/images/logo-api.gouv.fr-white.png"
          />
        </a>
        <ul
          className="jsx-696190668 footer__social"
        >
          <li
            className="jsx-696190668"
          >
            <a
              className="jsx-696190668"
              href="https://twitter.com/BetaGouv"
              title="Twitter"
            >
              <img
                alt="Twitter"
                className="jsx-696190668 icon icon-twitter"
                src="/static/images/social/twitter.svg"
              />
            </a>
          </li>
          <li
            className="jsx-696190668"
          >
            <a
              className="jsx-696190668"
              href="https://github.com/betagouv"
              title="Github"
            >
              <img
                alt="Github"
                className="jsx-696190668 icon icon-github"
                src="/static/images/social/github.svg"
              />
            </a>
          </li>
          <li
            className="jsx-696190668"
          >
            <a
              className="jsx-696190668"
              href="mailto:contact@particulier.api.gouv.fr"
              title="Nous écrire un mail"
            >
              <img
                alt="Email"
                className="jsx-696190668 icon icon-mail"
                src="/static/images/social/email.svg"
              />
            </a>
          </li>
        </ul>
      </div>
      <ul
        className="footer__links"
      >
        <li>
          <h2>
            particulier.api.gouv.fr
          </h2>
        </li>
        <li>
          <a
            href="https://status.particulier.api.gouv.fr"
          >
            Status des API
          </a>
        </li>
      </ul>
    </div>
  </footer>
</div>
`;
